tox.ini

```diff
@@ -0,0 +1,58 @@
+[tox]
+envlist = py26,py27
+
+[testenv]
+# ignore_errors makes sure all commands get run, which means this won't abort if flake8 fails.
+# it doesn't affect the return code in the end.
+ignore_errors=True
+setenv =
+    PYTHONDONTWRITEBYTECODE=1
+    PYTHONPATH={toxinidir}/vendor/lib/python
+deps =
+# test deps
+    certifi
+    coverage
+    flake8
+    mock
+    ordereddict
+    Paste
+    pyflakes
+    pytest
+    pytest-cov
+    requests
+# compiled deps
+    mysql-python==1.2.3
+    jinja2==2.5.5
+    sqlalchemy==0.7.1
+    simplejson==2.0.9
```

Comment:
I wonder is there a better way to specify these, so both `tox` and `python setup.py test` would install them... Not a blocker though